extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -207,7 +214,7 @@ export class ConsentUI {
         );
       }
       this.ui_ = dev().assertElement(promptElement);
-    } else if (promptUISrc) {
+    } else if (promptUISrc && !isStoryConsent) {
```

Comment:
I think it's better to apply the check at a much earlier stage maybe `validateAndParseConfig_()`.

Up to the story team, but I think we should treat this as a user error and use `userAssert()`